### PR TITLE
feat(insights): gates tab phase 1 (NPPD-1604)

### PR DESCRIPTION
**Pull request by kmwilkerson**
## Summary

Adds the Gates tab to the Insights wizard. Unlike Tabs 6 and 7 (which source entirely from local WooCommerce data), Gates is a behavioral funnel tab sourced from GA4 event data via BigQuery and local Woo for paid conversion completion.

This PR ships **Phase 1 only**: full UI with placeholder values (0, 0%, $0.00). All structure, sections, copy, and visualizations are final. Phase 2 (wiring real data via the BQ query proxy on Newspack Manager) lands separately once NPPD-1630 ships.

Gated behind `NEWSPACK_INSIGHTS_GATES_PREVIEW` constant so only environments with the constant defined see the tab during the placeholder phase.

Stacks on the Subscribers PR (#217, NPPD-1616), which stacks on the chrome PR (#211, NPPD-1602). Chrome and Subscribers need to merge first for Gates to be visible.

image

### Tab visibility (preview feature flag)

- `NEWSPACK_INSIGHTS_GATES_PREVIEW` constant gates Tab 4 rendering in the wizard tab nav
- Without the constant, Tab 4 doesn't render. Tab 6 and Tab 7 unaffected
- Independent from `NEWSPACK_INSIGHTS_ENABLED` so the placeholder phase doesn't leak to publishers

### Gates metric orchestrator with placeholder payloads

- `Gates_Metric` orchestrator with method stubs for every metric in the spec
- All methods return `{value: 0, computable: false, pending: true, placeholder_type: 'count'|'rate'|'currency'|'decimal'}` payloads
- REST endpoint `GET /newspack-insights/v1/gates` returns the full payload shape Phase 2 will populate
- Easy swap target for Phase 2: the SQL queries from `formulas/gates.md` plug in here, the UI doesn't change

### MetricCard `pending` state

- Adds `pending: true` handling to the shared `MetricCard` component
- When pending, renders the card with the placeholder value in normal styling (no greying, no per-card tooltip)
- Comparison mode deltas suppressed when pending (no real delta to show)
- Additive change. Tab 6 and Tab 7 metrics never set `pending: true`, so their rendering is unaffected

### React UI

- `GatesTab` with 5 sections following the behavioral-funnel layout:
  - **Gate exposure** (4 scorecards): Total Gate Impressions, Unique Readers Reached, Avg Exposures per Reader, Sessions With a Gate
  - **Free reader conversion** (2 scorecards): Regwall Conversion Direct, Regwall Conversion Influenced 7d
  - **Paid reader conversion** (4 scorecards): Paywall Conversion Direct, Paywall Conversion Influenced 14d, Total Paywall Revenue, Avg Revenue per Paywall Conversion
  - **How readers convert**: Conversion funnel (left) + Exposures-before-conversion distribution table (right)
  - **Performance by gate**: Per-gate breakdown table with empty-state row
- Top-of-tab preview banner explaining the placeholder phase, dismissable, reappears on page reload
- Direct vs Influenced explainer callout at the top of the tab (above Section 1), dismissable per session
- Date range picker, comparison toggle, and tab navigation all functional during Phase 1
- Section captions explain temporal scope, matching Tab 6 / Tab 7 styling

### Tab-local funnel and distribution components

- `Funnel.tsx` at `tabs/gates/viz/`: vertical, three-stage, renders stage label + value
- `Distribution.tsx` at `tabs/gates/viz/`: bucketed table (1 / 2 / 3-5 / 6+ exposures) with count and percentage columns
- Both built tab-local to keep blast radius small. When Tab 3 (Conversion Journey) needs the same patterns, these can be promoted to shared components in a future PR.

### Performance by gate table

- Empty-state row reading "No gate data yet. Performance metrics will appear once readers begin interacting with your gates."
- When Phase 2 populates real data, rows enrich with `wp_posts.post_title` server-side from `gate_post_id`
- Em-dash convention for non-applicable cells (gates without registration blocks show em-dash in Regwall columns, gates without checkout buttons show em-dash in Paywall columns), matching Tab 7's convention

## How to test

1. Set both `NEWSPACK_INSIGHTS_ENABLED` and `NEWSPACK_INSIGHTS_GATES_PREVIEW` constants in `wp-config.php`
2. Navigate to `wp-admin/admin.php?page=newspack-insights`
3. Click into the Gates tab
4. Verify:
   - [ ] Top-of-tab preview banner appears, dismissable via X, reappears on hard refresh
   - [ ] Direct vs Influenced explainer callout appears below the banner, above Section 1, dismissable
   - [ ] All 5 sections render in order: Gate exposure → Free reader conversion → Paid reader conversion → How readers convert → Performance by gate
   - [ ] Section 1: 4 scorecards in a row, all showing 0 / 0.0 / 0%
   - [ ] Section 2: 2 scorecards side-by-side, both showing 0%
   - [ ] Section 3: 4 scorecards in a row, showing 0% / 0% / $0.00 / $0.00
   - [ ] Section 4: Funnel renders vertically with three stages (Impression / Engagement / Conversion), all 0. Distribution table renders 4 buckets, all 0 / 0%
   - [ ] Section 5: Performance by gate table shows empty-state row
   - [ ] Section captions render under each section header
   - [ ] Date range picker changes don't error (values stay 0 since pending)
   - [ ] Comparison toggle on doesn't error (deltas don't render since pending)
5. Remove `NEWSPACK_INSIGHTS_GATES_PREVIEW` from `wp-config.php`, refresh → Tab 4 disappears from nav
6. Verify Tab 6 (Subscribers) and Tab 7 (Donors) still render normally with real data (regression check on the shared `MetricCard` pending state addition)

### CODE REVIEW

File: plugins/newspack-plugin/src/wizards/insights/tabs/GatesTab.tsx [hunk 1]
```diff
@@ -1,19 +1,76 @@
...
+
+const GatesTab = ( { range, previousRange }: GatesTabProps ) => {
+	const { status, data, error } = useGatesData( range, previousRange );
+
+	if ( status === 'loading' && ! data ) {
+		return (
+			<div className="newspack-insights__tab-loading" role="status" aria-live="polite">
+				{ __( 'Loading gate data…', 'newspack-plugin' ) }
+			</div>
+		);
```
[the remark this answers is not in the document]
Comment (kmwilkerson) [reply plugins/newspack-plugin/src/wizards/insights/tabs/GatesTab.tsx:47]: Fixed by cleaning up the branch. It had `main` accidentally merged in (cb6c7e1bfb), inflating the diff to 173 files. I rebuilt the branch as `nppd-1617` (donors) + the 6 real gates commits and force-pushed — the diff is now **20 files, all gates/insights**. The unrelated theme / My Account / reader-auth / CI / release changes are gone. The gates work itself is byte-for-byte identical (verified), and the pre-cleanup state is preserved at `backup/nppd-1604-pre-cleanup`.

File: plugins/newspack-plugin/src/blocks/reader-registration/index.php [hunk 2]
```diff
@@ -77,9 +77,8 @@ function enqueue_scripts() {
 		$handle,
 		'reader_registration_block_config',
 		[
-			'require_account_verification' => \Newspack\Content_Gate::requires_account_verification(),
-			'verification_url'             => \admin_url( 'admin-ajax.php' ),
-			'verification_nonce'           => \wp_create_nonce( 'newspack_reader_registration_verification' ),
+			'verification_url'   => \admin_url( 'admin-ajax.php' ),
+			'verification_nonce' => \wp_create_nonce( 'newspack_reader_registration_verification' ),
 		]
```
[the remark this answers is not in the document]
Comment (kmwilkerson) [reply plugins/newspack-plugin/src/blocks/reader-registration/index.php:82]: No longer in the diff after the branch cleanup. This `reader-registration` change came from `main` (the reader-auth unification, #135) via the accidental merge — it was never part of the gates work. If it's a genuine regression it belongs with @dkoo's reader-verification work, not this PR.

File: plugins/newspack-plugin/CHANGELOG.md [hunk 3]
```diff
@@ -1,4 +1,32 @@
-## [6.42.1](https://github.com/Automattic/newspack-plugin/compare/v6.42.0...v6.42.1) (2026-06-02)
+## newspack [6.42.4](https://github.com/Automattic/newspack-workspace/compare/newspack@6.42.3...newspack@6.42.4) (2026-06-04)
+
+
+### Bug Fixes
+
+* **integrations:** skip unconfigured + dedupe health-check alerts ([#207](https://github.com/Automattic/newspack-workspace/issues/207)) ([63eebb9](https://github.com/Automattic/newspack-workspace/commit/63eebb9a30b54f3c265db4be3584e69218239441))
```
[the remark this answers is not in the document]
Comment (kmwilkerson) [reply plugins/newspack-plugin/CHANGELOG.md:6]: Removed by the cleanup — these were auto-generated release CHANGELOG entries pulled in by the accidental `main` merge, not hand-edited here.

File: plugins/newspack-newsletters/CHANGELOG.md [hunk 4]
```diff
@@ -1,3 +1,10 @@
+## newspack-newsletters [3.33.6](https://github.com/Automattic/newspack-workspace/compare/newspack-newsletters@3.33.5...newspack-newsletters@3.33.6) (2026-06-03)
+
+
+### Bug Fixes
+
+* **posts-inserter:** show placeholder when block has zero posts ([#166](https://github.com/Automattic/newspack-workspace/issues/166)) ([c269ff2](https://github.com/Automattic/newspack-workspace/commit/c269ff2e9f937a406aaeae4a833da3046cbc61ab))
```
[the remark this answers is not in the document]
Comment (kmwilkerson) [reply plugins/newspack-newsletters/CHANGELOG.md:6]: Removed by the cleanup — same as above: release-automation CHANGELOG entry from the accidental `main` merge, no longer in the diff.
